modules/migration/schemas/comment.json

```diff
@@ -0,0 +1,61 @@
+{
+  "title": "Comment",
+  "description": "Comments received on issues associated to a repository within a forge (Gitea, GitLab, etc.).",
+
+  "type": "array",
+  "items": {
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+      "issue_index": {
+        "description": "Unique identifier, relative to the issue.",
+        "type": "number"
+      },
+      "poster_id": {
+        "description": "Unique identifier of the user who authored the comment.",
+        "type": "number"
+      },
+      "poster_name": {
+        "description": "Name of the user who authored the comment.",
+        "type": "string"
+      },
+      "poster_email": {
+        "description": "Email of the user who authored the comment.",
+        "type": "string"
+      },
+      "content": {
+        "description": "Long, multiline, description.",
```

Comment:
```suggestion
        "description": "Multiline comment.",
```

Reply:
Good catch, thanks! I copied stuff from the issue schema for consistency not sure how I missed this :)